src/core_test/mpas_test_core_halo_exch.F

```diff
@@ -249,6 +250,7 @@ subroutine test_core_halo_exch_full_test(domain, threadErrs, err)!{{{
 
       call test_core_halo_exch_validate_fields(domain, threadErrs, err)
 
+      call test_halo_adj_exch_fields(domain, threadErrs, err)
```

Comment:
Will this call overwrite any non-zero error flag that was returned by the `test_core_halo_exch_validate_fields` call above?